SabreTools.DatTools/Rebuilder.cs

```diff
@@ -226,7 +227,17 @@ public class Rebuilder
 
                     // If we are supposed to delete the file, do so
                     if (delete && rebuilt)
-                        File.Delete(input);
+                    {
+                            try 
```

Comment:
Please be consistent with indentation. I think you're using the GitHub editor since the new indent is 8 space characters compared to the 4 with the rest of the code.